benchbuild/projects/benchbuild/lapack.py

```diff
@@ -66,43 +71,44 @@ def compile(self):
                 ]
                 makefile.writelines(content)
 
-            _make = run.watch(make)
-            _make("-j", get_number_of_jobs(CFG), "f2clib", "blaslib")
-            with local.cwd(local.path("BLAS") / "TESTING"):
+            _make = bb.watch(make)
+            _make("-j", CFG["jobs"], "f2clib", "blaslib")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Value 'CFG' is unsubscriptable](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)